stacks/spica/packages/core/layout/home/home.layout.ts

```diff
@@ -19,7 +19,7 @@ export class HomeLayoutComponent implements OnInit {
 
   expanded = true;
 
-  routes$: Observable<Route[]>;
+  routes$: Observable<[Route[]]>;
```

Comment:
Can we use a single object to keep routes with their categories instead of using an array of route array?